src/main/java/org/breedinginsight/brapps/importer/model/imports/experimentObservation/ExperimentObservation.java

```diff
@@ -173,13 +174,17 @@ public BrAPIStudy constructBrAPIStudy(
         design.setPUI(designType);
         design.setDescription(designType);
         study.setExperimentalDesign(design);
-
+        String envSequenceValue = null;
+        if( commit ){
+            envSequenceValue = envNextVal.get().toString();
+        }
+        study.putAdditionalInfoItem( BrAPIAdditionalInfoFields.ENVIRONMENT_NUMBER, envSequenceValue);
```

Comment:
Should this be inside the `if(commit)` block?

Reply:
Done